cala-ledger-core-types/src/tx_template.rs

```diff
@@ -0,0 +1,74 @@
+use cala_cel_interpreter::{CelExpression, CelType, CelValue};
+use serde::{Deserialize, Serialize};
+
+use super::primitives::*;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct TxTemplateValues {
+    pub id: TxTemplateId,
+    pub code: String,
+    pub params: Option<Vec<ParamDefinition>>,
+    pub tx_input: TxInput,
+    pub entries: Vec<EntryInput>,
+    pub description: Option<String>,
+    pub metadata: Option<serde_json::Value>,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ParamDefinition {
+    pub name: String,
+    pub r#type: ParamDataType,
+    pub default: Option<CelExpression>,
+    pub description: Option<String>,
```

Comment:
In sqlx-ledger ```default``` is  ```Option<String>``` and we have a ```to_default_expr``` that converts ```Option<String>``` to Option<CelExpression```. Should I change it ````Option<String>``` or this works ?

Reply:
this type should have `Option<CelExpression>`. Its the builder that should be `Option<String>`